tests/compat/test_porcelain.py

```diff
@@ -91,3 +91,54 @@ def test_verify(self):
         tag = self.repo[b"refs/tags/verifyme"]
         self.assertNotEqual(tag.signature, None)
         tag.verify()
+
+
+@skipIf(
+    platform.python_implementation() == "PyPy" or sys.platform == "win32",
+    "gpgme not easily available or supported on Windows and PyPy",
+)
+class CommitCreateSignTestCase(PorcelainGpgTestCase, CompatTestCase):
+    def setUp(self):
```

Comment:
This setUp() is a noop, remove it?

Reply:
I'm not sure. I don't know Python multiple inheritance. I don't know what the default behavior is. I'm guessing `super()` only accesses the first parent class. Do you know?

You're thinking it's a noop because it just re-implements the default behavior?

Reply:
BTW, this setUp is a copy of the tag test case above it. I'm just copying the pattern of the existing test case for a tag, but re-adapting it to a commit.

Reply:
See https://docs.python.org/3/library/functions.html#super, I'm pretty sure dropping this is a noop.

I've fixed both of these things in main for the existing test; see https://github.com/jelmer/dulwich/pull/1439